src/components/impact-card/ImpactCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Flex, Heading, Text } from "@radix-ui/themes";
+import React from "react";
+
+interface ImpactCardProps {
+  icon: React.ReactNode;
+  value: string;
+  label: string;
+  className?: string;
+}
+
+const ImpactCard: React.FC<ImpactCardProps> = ({
+  icon,
+  value,
+  label,
+  className,
+}) => {
+  return (
+    <Box
+      width="100%"
+      height="248px"
+      p="10px"
+      className={`border border-[#12121226] bg-white rounded-xl ${className}`}
+    >
+      <Flex
+        width="100%"
+        height="100%"
+        className="bg-navy-200 rounded-lg p-10"
+        direction="column"
+        gap="8"
+        justify="end"
+      >
+        <Flex align="start" justify="center" gap="5">
+          {icon}
+          <Text
+            className="text-navy-900 text-[40px] lg:text-[65px]"
+            weight="light"
+          >
+            {value}
+          </Text>
+        </Flex>
+
+        <Heading
+          className="text-navy-900"
+          as="h2"
```

Comment:
You already have an H2 (“Our Impact”) in the main component. Inside each card, avoid using it again.
If the card’s label should be a heading, use <h3>. If it’s just a label, use <p>.
Purpose: keep a logical heading order; avoid H2 → H2 loops in screen readers